omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java

```diff
@@ -296,7 +301,10 @@ public Object getValue(Obs obs) throws ConversionException {
 			return null;
 		
 		if (obs.getValueDatetime() != null) {
-			return ConversionUtil.convert(obs.getValueDatetime(), Date.class);
```

Comment:
FWIW I think this line wasn't doing anything, basically saying "convert an Object of type Date to an object of type Date".